README.md

```diff
@@ -18,3 +18,11 @@ a library which processes themes created for
 | Environment | Wallpaper                                                                               | Colour scheme       | Fonts                | Full configuration                                                         |
 |-------------|-----------------------------------------------------------------------------------------|---------------------|----------------------|----------------------------------------------------------------------------|
 | GNOME 44    | [Atlantis Garbage Worker by Grady Frederick](https://www.artstation.com/artwork/W2GERJ) | Generated by Stylix | Cantarell, Fira Code | [See here](https://gist.github.com/danth/5e0b263eeed99aafcef2a2eca3e8548d) |
+
+![Hyprland](https://github.com/SomeGuyNamedMy/Screenshots/blob/60cd34c7c822042e33d0a1d7ae98a64290aa6dab/example1.png)
+![Hyprland](https://github.com/SomeGuyNamedMy/Screenshots/blob/60cd34c7c822042e33d0a1d7ae98a64290aa6dab/example2.png)
+![Hyprland](https://github.com/SomeGuyNamedMy/Screenshots/blob/60cd34c7c822042e33d0a1d7ae98a64290aa6dab/example3.png)
+
+| Environment | Wallpaper                                                                               | Colour scheme       | Fonts                | Full configuration                                                         |
+|-------------|-----------------------------------------------------------------------------------------|---------------------|----------------------|----------------------------------------------------------------------------|
+| Hyprland    | taken from raison ![d'etre by Eve]("https://www.youtube.com/watch?v=ulfY8WQE_HE")       | Generated by Stylix | DeJavu               | [See here]("https://github.com/SomeGuyNamedMy/users")                      |
```

Comment:
Assuming this was meant to be a link rather than an image, as currently it doesn't work.

```suggestion
| Hyprland    | Taken from [raison d'etre by Eve](https://www.youtube.com/watch?v=ulfY8WQE_HE)       | Generated by Stylix | DeJavu               | [See here](https://github.com/SomeGuyNamedMy/users)                      |
```

Reply:
yup, wups.